src/SSVOpenHexagon/Core/HGUpdate.cpp

```diff
@@ -134,6 +134,11 @@ void HexagonGame::update(ssvu::FT mFT)
 
             player.updatePosition(*this, mFT);
 
+            status.accumulateFrametime(mFT);
+            if (levelStatus.scoreOverridden) 
+            {
+                status.updateCustomScore(lua.readVariable<float>(levelStatus.scoreOverride));
+            }
```

Comment:
Was there any reason why `accumulateFrametime` was down below before? Honest question, can't remember...

Reply:
I dunno, but I think I made this change in the code because it was supposed to fix a small error in the level time when it saves to the menu (considering it was slightly off from the time shown in the attempt).